sechub-doc/src/docs/asciidoc/documents/operations/server/server_deployment.adoc

```diff
@@ -6,58 +6,36 @@
 {sechub} requires a PostgreSQL DB, Version > 10.x
 
 ===== Plain server
-In this scenario you would configure a database server available from {sechub}.
-
-_For example: Just install PostgreSQL DB server on same machine as your server._
+In this scenario you can just install a PostgreSQL DB instance on your {sechub} server machine.
 
 ===== Kubernetes
-Here you must provide a PostgreSQL DB server deployment *(one POD only)*,
-define a network policy, services, endpoints etc. etc.
-
-[IMPORTANT]
-====
-Be aware to mount *only ONE pod* - the database is *shared by all {sechub} server
-PODs* _(means this is a shared kernel / bottle neck but necessary)_
-====
+Here you need to deploy a PostgreSQL DB server in your Kubernetes namespace.
 
 [TIP]
 ====
-Think about an automated database backup - no matter which is your choosen
-installation variant...
+Think about regular database backups - no matter which variant you choose...
 ====
 
+
 ==== File upload storage
 
-{sechub} requires a file system folder where source scan uploads can be transferred to.
+{sechub} requires a shared file system folder to store uploads. +
+This can be a shared disk, a network share (e.g. NFS) or a S3 storage.
 
 [IMPORTANT]
 ====
-In a clustered environment like kubernetes the storage must be accessible by
+In a clustered environment like Kubernetes the storage must be accessible by
 every cluster member!
 ====
 
-===== Plain server
-In this scenario you would just either reuse {sechub} server file system or
-define a network share available form server.
-
-===== Kubernetes
-Here you must provide a PostgreSQL DB server deployment *(one POD only)*,
-define a network policy, services, endpoints etc.
-
-[IMPORTANT]
-====
-Be aware to mount *only ONE pod* - the database is *shared by all {sechub} server
-PODs* _(means this is a shared kernel / bottle neck but necessary)_
-====
 
 ==== Server Application
 
 {sechub} server is a written in `Java` and needs at least `{required_JDK}`.
 
-You can configure {sechub} server by system properties or also by  environment
-variables.
+You can configure the {sechub} server by system properties or also by environment variables.
 
-Next blocks describe necessary system properties and their ENV pendants.
+The next blocks describes necessary system properties and their environment variable pendants.
```

Comment:
```suggestion
The next block describes necessary system properties and their environment variable pendants.
```

Either remove the "s" from block, or from describe.
"The next block describes..."
"The next blocks describe..." 
depending on weather one or more blocks follow.

Reply:
:+1:   Grammar is fixed